src/d3d9/d3d9_device.cpp

```diff
@@ -4655,18 +4655,44 @@ namespace dxvk {
     // D3D9 doesn't have the concept of a framebuffer object,
     // so we'll just create a new one every time the render
     // target bindings are updated. Set up the attachments.
+    VkSampleCountFlagBits sampleCount;
+    VkExtent2D extent;
+    if (likely(!m_state.renderTargets[0]->IsNull())) {
+        const DxvkImageCreateInfo& rtImageInfo = m_state.renderTargets[0]->GetCommonTexture()->GetImage()->info();
+        sampleCount   = rtImageInfo.sampleCount;
+        extent.width  = rtImageInfo.extent.width;
+        extent.height = rtImageInfo.extent.height;
+    } else if (likely(m_state.depthStencil != nullptr)) {
+        const DxvkImageCreateInfo& dsImageInfo = m_state.depthStencil->GetCommonTexture()->GetImage()->info();
+        sampleCount   = dsImageInfo.sampleCount;
+        extent.width  = dsImageInfo.extent.width;
+        extent.height = dsImageInfo.extent.height;
+    } else {
+      Logger::err("RT0 and DS unbound");
```

Comment:
I don't think we need to log this case. Games can submit no-op draws in order to funnel things like RESZ to the driver and preserve ordering.

Reply:
Should I also do a RT0 nullptr check then? Nine for one doesn't. This case should never really be reached.

Or just remove the logging?

Reply:
RT0 being nullptr is not valid state ever, so I'd remove that else block entirely.

Reply:
This message is in case a game tries to use the NULL format without a depth stencil which should never happen.

Reply:
> Games can submit no-op draws in order to funnel things like RESZ to the driver and preserve ordering.

This doesn't matter to us, but games will still do it because d3d9 command submitting behaviour is silly. I don't want a bunch of log spam for no reason...